### PR TITLE
Add Support for Checkbox Questions

### LINKED ISSUE #186: Option for MCQ to be a Checkbox Question

### PR DESCRIPTION
**Pull request by carson-ricca**
Closes #186

### CODE REVIEW

File: course/models/models.py [hunk 1]
```diff
@@ -456,6 +459,13 @@ def submit(self):
...
     @property
     def answer_display(self):
+        if self.uqj.is_checkbox():
+            values = []
+            for key, value in self.uqj.get_rendered_choices().items():
+                for answer in self.answer.split(','):
+                    if key == answer:
+                        values.append(value)
+            return values
         return self.uqj.get_rendered_choices().get(self.answer, 'Unknown')
```
Comment (keyvankhademi) [inline course/models/models.py:469]:
```suggestion
    @property
    def answer_display(self):
        values = []
        rendered_choices = self.uqj.get_rendered_choices()
        for answer in self.answer.split(','):
            values.append(rendered_choices.get(answer, 'Unknown'))
        return values  
```
We can't have a function that returns 2 different types of values.
Also, I'm not a fan of nested for loops in this case.
It also ignores the answers that are not in the choices. We should somehow specify something weird has happened.

File: course/views/views.py [hunk 2]
```diff
@@ -69,9 +58,6 @@ def question_view(request, pk, key=None):
     if isinstance(question, JavaQuestion):
         return _java_question_view(request, question, key)
 
-    if isinstance(question, CheckboxQuestion):
-        return _multiple_choice_question_view(request, question, 'checkbox_question.html', key)
```
Comment (keyvankhademi) [inline course/views/views.py:73]: The html files that are not being used anymore should also be deleted.

**Review by keyvankhademi — CHANGES_REQUESTED** [review]

File: course/views/views.py (hunk 2 above)
Comment (carson-ricca) [reply course/views/views.py:73]: Do I delete just the html files or do I delete all the rendering related to problems?

File: course/grader/grader.py [hunk 3]
```diff
@@ -17,14 +17,19 @@ def grade(self, submission):
 class MultipleChoiceGrader(Grader):
 
     def grade(self, submission):
-        if submission.answer != submission.question.answer:
-            return False, 0
+        correct_count = 0
```
Comment (carson-ricca) [inline course/grader/grader.py:20]: A basic implementation that supports the grading of checkbox questions. Needs requirement clarifications then improvements can be made.

**Review by keyvankhademi — APPROVED** [review]

**Pull request merged**